packages/components/psammead-promo-separator/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import { C_LUNAR } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+const PromoSeparator = styled.hr`
+  color: ${C_LUNAR};
+  margin: ${GEL_SPACING} 0 ${GEL_SPACING_DBL};
+  height: 0.0625rem;
```

Comment:
I wouldn't expect something that's designed as 1px thick to be scaled with REM, could we confirm this with UX and from an accessibility POV 
cc @greenc05

Reply:
```suggestion
  height: 1px;
```

Reply:
I actually wanted to define it as PX, but since we mostly define everything with REM, I put it like that. I will double check this with them.